### PR TITLE
Minor Borg Qol

### PR DESCRIPTION
**Pull request by Emisse**
planning on doing more stuff later but wanted to do some easy qol stuff for them since they are bloopy
basically:
-they cant slip anymore
-they all have intrinsic radio for common,sci,binary and can also hear/speak on cooresponding dept radio (restricting their radio access just makes it a bit boring)
-they can be locked by people with access so tiders cant pull their brains out 
-they are 50% tankier since they are very squishy atm
-department specific borgs start with basic kits for their departments so they arent left hanging with nothing
-buffed many of the lackluster modules with applicable/qol tools

🆑 
- tweak: Borgs have had a QOL pass, they are access locked now and the department specific ones start with applicable tools.

### CODE REVIEW

**Comment by GoodWheatley** [comment]
Emagging a locked borg panel should probably remove the access requirement to unlock it, without that it’s going to be pretty hard to subvert a borg.

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml [hunk 1]
```diff
@@ -455,3 +472,5 @@
     items:
     - BikeHorn
     - ClownRecorder
+    - CrayonRainbow
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:475]: feed or use up the crayon for esword slot???

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml [hunk 2]
```diff
@@ -397,6 +409,9 @@
     items:
     - Pen
     - BooksBag
+    - HandLabeler
+    - Lighter
+    - DrinkShaker
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:414]: what

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by EmoGarbage404** [comment]
I dont remember exactly what the cost for each module is, but it might be best to just move the basic department specific ones out of research if this is the way we want to go with it. You can then also adjust the cost to make them in turn so its less expensive for basic kitting.

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml (hunk 2 above)
Comment (CatMagic1) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:414]: Hand labeler makes sense, lighter is for... arson? Burning papers. :trollface: 

Shaker should probably be in it's own bartending module
but that will probably just take the bartenders job, but also it seems fun for the service borg to be able hold reagents and transfer reagents so yeah I guess

**Comment by deltanedas** [comment]
also silently buffed borgs to 4 modules :trollface:

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml (hunk 2 above)
Comment (lapatison) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:414]: But how do you pour something into that shaker?

Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:414]: add a syringe to the *literacy* module :trollface:

**Comment by EmoGarbage404** [comment]
> also silently buffed borgs to 4 modules :trollface:

Idc much about that but regular borg should get buffed to 6 or 7 to compensate and not powercreep it.

**Comment by deltanedas** [comment]
yeah but specific borgs having 4 modules means generally they can have every specific module + tools meaning theres no more need to think about what you want a borg to do, just have everything :trollface:

**Comment by Doru991** [comment]
Should probably update the mob threshold component too so 'Dead' is at 150, otherwise the health indicator will go red too early

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml (hunk 1 above)
Comment (Emisse) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:475]: ah yea i forgot it disappears when it gets eat

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml (hunk 2 above)
Comment (Emisse) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:414]: i forgor to rename it to service module

**Comment by melt4mePastyCakes** [comment]
medical cyborg comes with treatment module but not diagnosis module, which seems weird since you can't properly treat someone without knowing what's wrong with them first

**Comment by Emisse** [comment]
they rely on doctors to diagnose first unless science builds them the module

**Comment by LankLTE** [comment]
> medical cyborg comes with treatment module but not diagnosis module, which seems weird since you can't properly treat someone without knowing what's wrong with them first

You can manually inspect people to see (vaguely) what's wrong with them too.

**Pull request merged**